### ISSUE TITLE
Video sometimes restarts after postroll

### ISSUE THREAD

**Issue by ddunkin** [videojs/videojs-contrib-ads#159]
The content video sometimes restarts after a postroll ad ends. This seems to happen more on high-latency connections, so I suspect it's some race. I have reproduced it using Chrome's network throttling on the _Regular 3G_ setting, but not consistently.

video.js 5.9.2
videojs-contrib-ads 3.2.0
Chrome 50.0
OS X 10.11.4

Here's the output from the example ad integration. Note the events after the first _ended_ event.

```
[09:26:50.168] [content-playback*] ended
[09:26:50.168] [content-playback*] contentplayback
[09:26:50.167] [content-resuming*] adtimeout
[09:26:50.065] [content-playback]  contentended
[09:26:50.064] [content-playback*] pause
[09:26:03.898] [content-playback*] ended
[09:26:03.898] [content-playback*] contentplayback
[09:26:03.377] [content-resuming]  contentcanplaythrough
[09:26:03.377] [content-resuming]  contentplaying
[09:26:03.375] [content-resuming]  contentcanplay
[09:26:03.375] [content-resuming]  contentseeked
[09:26:03.245] [content-resuming]  contentseeking
[09:26:02.981] [content-resuming]  contentseeking
[09:26:02.980] [content-resuming]  contentcanplaythrough
[09:26:02.980] [content-resuming]  contentplaying
[09:26:02.980] [content-resuming]  contentplay
[09:26:02.978] [content-resuming]  contentcanplay
[09:26:02.977] [content-resuming]  contentloadeddata
[09:26:02.916] [content-resuming]  contentloadedmetadata
[09:26:02.915] [content-resuming]  contentdurationchange
[09:26:02.903] [content-resuming]  contentloadstart
[09:26:02.897] [content-resuming*] adend
[09:26:02.894] [ad-playback]       adended
[09:26:02.894] [ad-playback]       adpause
[09:25:52.835] [ad-playback]       adcanplaythrough
[09:25:52.835] [ad-playback]       adplaying
[09:25:52.835] [ad-playback]       adcanplay
[09:25:52.834] [ad-playback]       adloadeddata
[09:25:52.766] [ad-playback]       adloadedmetadata
[09:25:52.765] [ad-playback]       addurationchange
[09:25:52.650] [ad-playback]       adloadstart
[09:25:52.645] [ad-playback]       adwaiting
[09:25:52.645] [ad-playback]       adplay
[09:25:52.644] [ad-playback]       adabort
[09:25:52.638] [content-playback]  contentended
[09:25:52.637] [ad-playback*]      adstart
[09:25:52.635] [content-playback*] pause
[09:22:57.098] [content-playback*] canplaythrough
[09:22:57.098] [content-playback*] playing
[09:22:57.098] [content-playback*] canplay
[09:22:57.098] [content-playback*] seeked
[09:22:56.870] [content-playback*] waiting
[09:22:56.870] [content-playback*] seeking
[09:22:56.870] [content-playback*] canplaythrough
[09:22:56.869] [content-playback*] playing
[09:22:56.868] [content-playback*] contentplayback
[09:22:56.867] [content-resuming]  contentplay
[09:22:56.865] [content-resuming]  contentcanplay
[09:22:56.864] [content-resuming]  contentloadeddata
[09:22:56.786] [content-resuming]  contentloadedmetadata
[09:22:56.785] [content-resuming]  contentdurationchange
[09:22:56.774] [content-resuming]  contentloadstart
[09:22:56.769] [content-resuming*] adend
[09:22:56.766] [ad-playback]       adended
[09:22:56.766] [ad-playback]       adpause
[09:22:46.695] [ad-playback]       adcanplaythrough
[09:22:46.694] [ad-playback]       adplaying
[09:22:46.693] [ad-playback]       adcanplay
[09:22:46.692] [ad-playback]       adloadeddata
[09:22:46.635] [ad-playback]       adloadedmetadata
[09:22:46.635] [ad-playback]       addurationchange
[09:22:46.511] [ad-playback]       adloadstart
[09:22:46.507] [ad-playback]       adwaiting
[09:22:46.507] [ad-playback]       adplay
[09:22:46.506] [ad-playback]       adabort
[09:22:46.500] [ad-playback*]      adstart
[09:21:37.876] [content-playback*] canplaythrough
[09:21:37.876] [content-playback*] playing
[09:21:37.876] [content-playback*] canplay
[09:21:37.876] [content-playback*] seeked
[09:21:37.752] [content-playback*] waiting
[09:21:37.752] [content-playback*] seeking
[09:21:37.752] [content-playback*] canplaythrough
[09:21:37.751] [content-playback*] playing
[09:21:37.751] [content-playback*] contentplayback
[09:21:37.751] [content-resuming]  contentplay
[09:21:37.748] [content-resuming]  contentcanplay
[09:21:37.747] [content-resuming]  contentloadeddata
[09:21:37.722] [content-resuming]  contentloadedmetadata
[09:21:37.721] [content-resuming]  contentdurationchange
[09:21:37.062] [content-resuming]  contentloadstart
[09:21:37.057] [content-resuming*] adend
[09:21:37.053] [ad-playback]       adended
[09:21:37.053] [ad-playback]       adpause
[09:21:31.391] [ad-playback]       adcanplaythrough
[09:21:31.391] [ad-playback]       adplaying
[09:21:31.391] [ad-playback]       adcanplay
[09:21:31.390] [ad-playback]       adloadeddata
[09:21:31.306] [ad-playback]       adloadedmetadata
[09:21:31.306] [ad-playback]       addurationchange
[09:21:31.092] [ad-playback]       adloadstart
[09:21:31.084] [ad-playback]       adwaiting
[09:21:31.084] [ad-playback]       adplay
[09:21:31.082] [ad-playback]       adabort
[09:21:31.079] [ad-playback*]      adsready
[09:21:31.079] [ad-playback*]      adstart
[09:21:30.974] [ads-ready?*]       pause
[09:21:30.970] [ads-ready?*]       loadstart
```

**Comment by incompl** [comment]
I noticed another person reported this issue in #160. I have been trying to reproduce this issue so I can help find a resolution.

Are you also calling `endLinearAdMode` in `contentended` rather than `adended`? Are you able to reproduce your issue in the [example integration](https://github.com/videojs/videojs-contrib-ads/blob/master/example/example-integration.js)? Knowing this would help pinpoint the issue. Thanks!

**Comment by incompl** [comment]
I believe this was fixed in version 3.3.10. Please reopen or create a new issue if seen again.
